pkg/controller/disruption/disruption.go

```diff
@@ -43,6 +44,12 @@ import (
 
 const statusUpdateRetries = 2
 
+// DeletionTimeout sets maximum time from the moment a pod is added to DisruptedPods in PDB.Status
+// to the time when the pod is expected to be marked for deletion. If the pod was not marked for
+// deletion during that time it is assumed that it won't be deleted at all and the corresponding
+// entry can be removed from pdb.Status.DisruptedPods.
+const DeletionTimeout = 60 * time.Second
```

Comment:
Probably not. This is related to watch propagation latency. If this anywhere close to 1 min then we will have much bigger problems.